src/games/strategy/engine/framework/mapDownload/MapDownloadController.java

```diff
@@ -24,9 +23,8 @@
   private static final String TRIPLEA_LAST_CHECK_FOR_MAP_UPDATES = "triplea.lastCheckForMapUpdates";
   private final MapListingSource mapDownloadProperties;
 
-  public MapDownloadController() {
```

Comment:
Now the MapDownlaodController does not know anything about config file parsing : )